hack/update-codegen.sh

```diff
@@ -21,7 +21,15 @@ set -o pipefail
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
 
+CODEGEN_PKG="../vendor/k8s.io/code-generator"
+
 bash -x ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
-  github.com/argoproj/argo-events/pkg/client github.com/argoproj/argo-events/pkg/apis \
+  github.com/argoproj/argo-events/pkg/sensor-client github.com/argoproj/argo-events/pkg/apis \
```

Comment:
we should be using `github.com/argoproj/argo-events/pkg/client` for both

Reply:
Agreed.